validphys2/src/validphys/process_options.py

```diff
@@ -157,39 +155,48 @@ def _dijets_xq2map(kin_info):
 
 def _hqp_yq_xq2map(kin_info):
     # Compute x, Q2
-    if {"k1", "k2", "k3"} <= kin_info.keys():
-        kin_info[_Vars.y_t] = kin_info["k1"]
-        kin_info[_Vars.m_t2] = kin_info["k2"]
-        kin_info[_Vars.sqrts] = kin_info["k3"]
-
-    mass2 = kin_info.get_one_of(_Vars.m_t2, _Vars.m_ttBar)
-
+    # Theory predictions computed with HT/4 see 1906.06535
```

Comment:
```suggestion
    # Theory predictions computed with HT/4 ~ mt/2 for rapidity distr.
    # see section 3 from 1906.06535
```

so that if someone wants to source where this comes from is easier to find